python/ray/data/_internal/actor_autoscaler/default_actor_autoscaler.py

```diff
@@ -89,14 +90,36 @@ def _derive_target_scaling_config(
                     reason="operator exceeding resource quota"
                 )
             budget = self._resource_manager.get_budget(op)
-            if _get_max_scale_up(actor_pool, budget) == 0:
+            max_scale_up = _get_max_scale_up(actor_pool, budget)
+            if max_scale_up == 0:
                 return ActorPoolScalingRequest.no_op(reason="exceeded resource limits")
 
+            current_size = actor_pool.current_size()
+            # Calculate desired delta based on utilization
+            if current_size > 0 and util > 0:
+                plan_delta = math.ceil(
+                    current_size * (util / self._actor_pool_scaling_up_threshold - 1)
+                )
+            else:
+                plan_delta = 1
```

Comment:
Since we're already doing `max(1, delta)`, is it equivalent to just do this?
```suggestion
            plan_delta = math.ceil(
                current_size * (util / self._actor_pool_scaling_up_threshold - 1)
            )
```